ci/scripts/run-verilator-tests.sh

```diff
@@ -28,4 +28,5 @@ xargs ci/bazelisk.sh test \
     //sw/device/silicon_creator/lib/drivers:retention_sram_functest_sim_verilator \
     //sw/device/silicon_creator/lib/drivers:alert_functest_sim_verilator \
     //sw/device/silicon_creator/lib/drivers:watchdog_functest_sim_verilator \
-    //sw/device/silicon_creator/lib:irq_asm_functest_sim_verilator
+    //sw/device/silicon_creator/lib:irq_asm_functest_sim_verilator \
+    //sw/device/silicon_creator/mask_rom:mask_rom_epmp_test_sim_verilator \
```

Comment:
do you want the `\` on the last line?
```suggestion
    //sw/device/silicon_creator/mask_rom:mask_rom_epmp_test_sim_verilator
```